UICatalog/Scenarios/TabViewExample.cs

```diff
@@ -0,0 +1,210 @@
+	[ScenarioCategory ("Controls")]
+	class TabViewExample : Scenario {
+
+		TabView tabView;
+
+		MenuItem miShowTopLine;
+		MenuItem miShowBorder;
+		MenuItem miTabsOnBottom;
+
+		public override void Setup ()
+		{
+			Win.Title = this.GetName ();
+			Win.Y = 1; // menu
+			Win.Height = Dim.Fill (1); // status bar
+			Top.LayoutSubviews ();
+
+			var menu = new MenuBar (new MenuBarItem [] {
+				new MenuBarItem ("_File", new MenuItem [] {
+
+					new MenuItem ("_Add Blank Tab", "", () => AddBlankTab()),
+
+					new MenuItem ("_Clear SelectedTab", "", () => tabView.SelectedTab=null),
+					new MenuItem ("_Quit", "", () => Quit()),
+				}),
+				new MenuBarItem ("_View", new MenuItem [] {
+					miShowTopLine = new MenuItem ("_Show Top Line", "", () => ShowTopLine()){
+						Checked = true,
+						CheckType = MenuItemCheckStyle.Checked
+					},
+					miShowBorder = new MenuItem ("_Show Border", "", () => ShowBorder()){
+						Checked = true,
+						CheckType = MenuItemCheckStyle.Checked
+					},
+					miTabsOnBottom = new MenuItem ("_Tabs On Bottom", "", () => SetTabsOnBottom()){
+						Checked = false,
+						CheckType = MenuItemCheckStyle.Checked
+					}
+
+					})
+				});
+			Top.Add (menu);
+
+			tabView = new TabView () {
+				X = 0,
+				Y = 0,
+				Width = 60,
+				Height = 20,
+			};
+
+
+			tabView.AddTab (new Tab ("Tab1", new Label ("hodor!")), false);
+			tabView.AddTab (new Tab ("Tab2", new Label ("durdur")), false);
+			tabView.AddTab (new Tab ("Interactive Tab", GetInteractiveTab ()), false);
+			tabView.AddTab (new Tab ("Big Text", GetBigTextFileTab ()), false);
+			tabView.AddTab (new Tab (
+				"Long name Tab, I mean seriously long.  Like you would not believe how long this tab's name is its just too much really woooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooowwww thats long",
+				 new Label ("This tab has a very long name which should be truncated.  See TabView.MaxTabTextWidth")),
+				 false);
+			tabView.AddTab (new Tab ("Les Mise" + Char.ConvertFromUtf32 (Int32.Parse ("0301", NumberStyles.HexNumber)) + "rables", new Label ("This tab name is unicode")), false);
+
+			for (int i = 0; i < 100; i++) {
+				tabView.AddTab (new Tab ($"Tab{i}", new Label ($"Welcome to tab {i}")), false);
+			}
+
+			tabView.SelectedTab = tabView.Tabs.First ();
+
+			Win.Add (tabView);
+
+			var frameRight = new FrameView ("About") {
+				X = Pos.Right (tabView),
+				Y = 0,
+				Width = Dim.Fill (),
+				Height = Dim.Fill (),
+			};
+
+
+			frameRight.Add (new TextView () {
+				Text = @"This demos the tabs control
+Switch between tabs using cursor keys
+
+",
+				Width = Dim.Fill (),
+				Height = Dim.Fill ()
+			});
+
+			Win.Add (frameRight);
+
+
+
+			var frameBelow = new FrameView ("Bottom Frame") {
+				X = 0,
+				Y = Pos.Bottom (tabView),
+				Width = tabView.Width,
+				Height = Dim.Fill (),
+			};
+
+
+			frameBelow.Add (new TextView () {
+				Text = @"This frame exists to check you can still tab here
```

Comment:
Same here (`her`) is displayed insetead of (`here`).

Reply:
Think this was down to using `@""` notation with newlines in source.  I fixed by changing to regular string `"some\ntext"`